admin/main/src/main/java/tech/aurora/bfadmin/controller/AdminController.java

```diff
@@ -43,22 +48,39 @@ public String getMainApp() {
   
   @RequestMapping("/dashboard")
   public String getDashboard(Model model) {
-    model.addAttribute("clusterInfo", clusterInfo);
-    model.addAttribute("clusterDetails", adminService.getClusterDetails());
-    model.addAttribute("awsRegion", region);
-    return "dashboard";
+    if (ui) {
+      generateSecurityKey();
+      model.addAttribute("clusterInfo", clusterInfo);
+      model.addAttribute("clusterDetails", adminService.getClusterDetails());
+      model.addAttribute("awsRegion", region);
+      model.addAttribute("securityKey", SecurityConfig.securityKey);
+      return "dashboard";
+    } else {
+      model.addAttribute("status", "999");
```

Comment:
is that http status?  you could use a common one

Reply:
Nope just a custom error code/message set to be displayed on the UI. It's arbitrary.